.nvmrc

```diff
@@ -0,0 +1 @@
+lts/*
```

Comment:
Is this a good practice? Maybe it would be better to have a fixed version of node here (at least a fixed major version).

Reply:
Fixed